etcdctl/README.md

```diff
@@ -344,13 +344,16 @@ LEASE TIMETOLIVE retrieves the lease information with the given lease ID.
 # lease 2d8257079fa1bc0c granted with TTL(500s)
 
 ./etcdctl put foo1 bar --lease=2d8257079fa1bc0c
+#OK
+
 ./etcdctl put foo2 bar --lease=2d8257079fa1bc0c
+#OK
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c
 # lease 2d8257079fa1bc0c granted with TTL(500s), remaining(481s)
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c --keys
-# lease 2d8257079fa1bc0c granted with TTL(500s), remaining(472s), attached keys([foo2 foo1])
```

Comment:
@gyuho I had one query. Actually i have just modified the way they are displayed and i haven't removed it, just re-ordered since i was getting the result in order of the keys they were created. So just wanted to confirm if you still wanted to change it back to the was it was? since its returing the order in sorted manner.
Request you to feedback on this. Will modify if you feel that way.
Thank you.